.github/workflows/publish.yml

```diff
@@ -15,5 +15,20 @@ jobs:
       - uses: actions/checkout@v3.3.0
       - name: Check that the image builds
         run: docker build . --file Dockerfile
+  validate-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.3.0
+      - name: Validate that action points to main branch
+        run: |
+          BRANCH=$(yq '.jobs.review-approvals.steps[0].uses' $FILE_NAME | cut -d "@" -f2)
+          # If the branch is not the main branch
+          if [ "$BRANCH" != "$MAIN_BRANCH" ]; then
```

Comment:
is it the contributor's PR branch destination? why can't it use other than a main branch?
i saw some PRs could go to some temp epic branch, and that would lead to PR being blocked by this action fail?

Reply:
I think this is supposed to be a helper check to be used only in this (`paritytech/review-bot`) repository, nowhere else.

Reply:
Yeah, it is as @rzadp has said. Because I need to experiment with a new feature I usually change where the action is obtaining the code to the PR's branch and sometimes I forget to change it and I have to do something like this: https://github.com/paritytech/review-bot/pull/36